tiempo/conn.py

```diff
@@ -20,10 +21,15 @@ def subscribe_to_backend_notifications(db=REDIS_QUEUE_DB):
 
 def hear_from_backend():
     events = []
-    while True:
-        message = NOTIFY_PUBSUB.parse_response(block=False)
+    def parse_backend():
```

Comment:
It's not immediately obvious to me what the advantage is of adding this closure. Maybe a comment?